WTC_Libs/stateMachine/comms.c

```diff
@@ -94,9 +94,9 @@ void commsFromGround(uint8_t* buffer) {
 	 */
 	char prompt[100] = { };
 	char promptToGround[100] = { };
-	commsRx->designator = fromBinary((uint8_t*)DMABUFFER[1]);
+	commsRx->designator = fromBinary((uint8_t*)(DMABUFFER + 1));
```

Comment:
`fromBinary` takes in a pointer. However, accessing an array with the bracket operators automatically dereferences the array offset and returns a value. Casting it back to a pointer creates a pointer from a value which could be anything. Attempting to dereference this value, which could be anything, including 0 (a null-pointer) could lead to problems. This offset handles pointer to `DMABUFFER[1]`. I've realized after the fact that more technically correct would be `DMABUFFER + (sizeof(DMABUFFER[0]) * 1)` (sizeof the first value of DMABUFFER, in this case `char`), however since `sizeof char` == 1 this still works.